tests/emscripten_log/emscripten_log.cpp

```diff
@@ -80,7 +80,12 @@ void __attribute__((noinline)) bar(int = 0, char * = 0, double = 0) {
     but the line numbers will greatly vary depending on the mode we are compiling in, so cannot test with direct string comparison. */
 
   if ((flags & EM_LOG_C_STACK) != 0) {
+    // TODO(https://github.com/emscripten-core/emscripten/issues/13089)
+    // We should be able to check for emscripten_log.cpp here but sadly
```

Comment:
from the linked bug, this was always an issue, and not uncovered by this PR's adding `-g` to libraries? if so could it be a separate PR? I'm not actually suggesting it be, this looks fine, but I'd like to understand if this has to be here or not.

Reply:
The bug already existed but this PR makes it show up.   The problem is that the source map seems to be re-read and when there is only on file in it it shows `emscripten_log.cpp` but when there are other files in the source map it shows them (and now `emscripten_log.cpp`) in a way that I don't understand, and is not correct either before or after this change.